packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -453,6 +453,7 @@ export default class NewVm extends BaseComponent {
       bootAfterCreate: state.bootAfterCreate,
       copyHostBiosStrings:
         state.hvmBootFirmware !== 'uefi' && !this._templateHasBiosStrings() && state.copyHostBiosStrings,
+      secureBoot: state.secureBoot,
```

Comment:
Is it ok to set the `secureBoot` on a non UEFI VM?

Reply:
No, secure boot only works in UEFI guests.